test/summit/seed.toml

```diff
@@ -22,3 +22,4 @@ priority = 20
 name = "recipes"
 summary = "Test recipes"
 uri = "https://github.com/AerynOS/recipes.git"
+branch = "2025-05-repo-rebuild"
```

Comment:
Was this meant to be committed? Are we still using this branch??

Reply:
Yeah its needed or testing via `just up` doesn't work